lib/common.sh

```diff
@@ -17,7 +17,20 @@ detect_sbt() {
 }
 
 is_play() {
```

Comment:
Do we need a new Play 3 test fixture?

Reply:
Generally, yes. I plan to update the getting started guide to Play 3 which will get us a smoke test for Play 3. For the issue that prompted this PR, the case is so rare I don't think adding an integration is worth it (sbt integration tests are really slow).